utils/ort/src/main/kotlin/VcsUtils.kt

```diff
@@ -98,3 +98,34 @@ fun normalizeVcsUrl(vcsUrl: String): String {
 
     return url
 }
+
+/**
+ * Return a browsable homepage URL for the given [vcsUrl], or the original URL
+ * if the host is unknown.
+ */
+fun toHomepageUrl(vcsUrl: String): String {
```

Comment:
We already have the [`VcsHost` enum class](https://github.com/oss-review-toolkit/ort/blob/60316b797569f05fc7835f6e2f3b2bb72340f71b/downloader/src/main/kotlin/VcsHost.kt) for basically this. Please consider using / extending it.

I guess something like `VcsHost.fromUrl(vcsUrl)` and then calling any of the `to*` function on the result should get you started.